brainvisa/toolboxes/spm/processes/spm12/util/SPM12composition_generic.py

```diff
@@ -58,13 +61,13 @@ def deformation_type_choice(num):
 
 def dartel_params(num):
     return {
-        f'flow_field_{num}': ReadDiskItem('4D Volume', getAllFormats(),
+        f'flow_field_{num}': ReadDiskItem('4D Volume', ['NIFTI-1 image', 'SPM image', 'MINC image'],
```

Comment:
You changed getAllFormats() for fields but you kept it for matrices, but the process will not work if you put something ohter than a .mat, no ?

Reply:
You are right, I kept it as generic as possible, but as you mentioned it might be useful to limit it to a mat file.